client/src/screens/Client/Home/components/modal/MessageToPatient.js

```diff
@@ -18,7 +18,7 @@ import clsx from "clsx";
 import IconButton from "@material-ui/core/IconButton";
 import CloseIcon from "@material-ui/icons/Close";
 import { makeStyles } from "@material-ui/core/styles";
-import Messages from "./../../../../../services/message-to-patient.service";
+// import Messages from "./../../../../../services/message-to-patient.service";
```

Comment:
This component was imported but not being used anywhere. So instead of removing the extra import, I just commented it out in case the developer wants to use it again in the future.

Reply:
okay.